lib/index.js

```diff
@@ -41,6 +41,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
 });
 
 /**
+ * Check for the existence of an identity.
+ *
+ * @param id the ID of the identity to check.
+ * @param callback(err, exists) called once the operation completes.
+ */
+api.exists = function(id, callback) {
+  database.collections.identity.find({
```

Comment:
Not sure if we need a permission check in here or not... @davidlehn, @mattcollier ... thoughts on checking `ACCESS` permission with this method? It's true that current usage will pass `null` for the actor and it will get skipped, but we probably want to allow checking on that, don't we?